cou/steps/plan.py

```diff
@@ -116,7 +116,7 @@ def verify_data_plane_ready_to_upgrade(args: CLIargs, analysis_result: Analysis)
     :type analysis_result: Analysis
     :raises DataPlaneCannotUpgrade: When data-plane is not ready to upgrade.
     """
-    if args.is_data_plane_command:
+    if args.scope is UpgradeScope.DATA_PLANE:
```

Comment:
@aieri we mention enums yesterday, but this do not make sense to me. Why not using `args.upgrade_group == DATA_PLANE`?

Reply:
Because there is also the `hypervisors` that is considered as data-plane